locales/en/messages.json

```diff
@@ -857,6 +857,10 @@
         "message": "A configuration change requires a reboot",
         "description": "Description of the REBOOT_REQD arming disable flag"
     },
+    "initialSetupArmingDisableFlagsTooltipFILTERING": {
+        "message": "A filtering setup error or unsafe configuration detected",
+        "description": "Description of the FILTERING arming disable flag"
```

Comment:
I made changes in #1526 it should be:

"description": "Message that pops up to describe the FILTERING arming disable flag"